modules/light-clients/06-solomachine/client_state_test.go

```diff
@@ -603,6 +605,35 @@ func (suite *SoloMachineTestSuite) TestVerifyMembership() {
 	}
 }
 
+func (suite *SoloMachineTestSuite) TestSignBytesMarshalling() {
+	for _, sm := range []*ibctesting.Solomachine{suite.solomachine, suite.solomachineMulti} {
```

Comment:
This can be removed. The for loop here is used to test a regular solo machine (single private key) and a multi sig solo machine (multiple private keys). Since there is no signing here, we can just run the test with the single private key solo machine